homeassistant/components/droplet/__init__.py

```diff
@@ -0,0 +1,65 @@
+"""The Droplet integration."""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .coordinator import DropletConfigEntry, DropletDataCoordinator
+
+PLATFORMS: list[Platform] = [
+    Platform.SENSOR,
+]
+
+logger = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, config_entry: DropletConfigEntry
+) -> bool:
+    """Set up Droplet from a config entry."""
+
+    if TYPE_CHECKING:
+        assert config_entry.unique_id is not None
+    droplet_coordinator = DropletDataCoordinator(hass, config_entry)
+
```

Comment:
```suggestion
    await droplet_coordinator.async_config_entry_first_refresh()
```
let's refresh the coordinator once before adding it